### PR TITLE
some general doc cleanup

### PR DESCRIPTION
**Pull request by sreynen**
This is an attempt to make the project a little more approachable to new contributors.

### CODE REVIEW

File: docs/HOWTO--server-backend.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# How to Develop with a Server Backend
+
+When running locally, you should set the following environment variables:
+
+```
+export API_URI="https://petitions.example.com"
+export BASE_APP_PATH="/Users/yourusername/Sites/mop-frontend/local/"
```
Comment (sandramchung) [inline docs/HOWTO--server-backend.md]: This should be generic-ized to something like `export BASE_APP_PATH="/path/to/your/local/mop-frontend/local/"`

File: docs/HOWTO--server-backend.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+export STATIC_ROOT="../../build/"
+```
+
+`API_URI` can either point at a hosted version of the API (as in the example), or a local instance of mop (something like `http://0.0.0.0:8000`).
```
Comment (sandramchung) [inline docs/HOWTO--server-backend.md]: They don't have access to mop yet. Can they use broom.moveon.org? Do we note that here or mention it to non-MO devs as needed?

Comment (shakalee14) [reply docs/HOWTO--server-backend.md]: same question

Comment (sreynen) [reply docs/HOWTO--server-backend.md]: I was thinking mention as needed.

Comment (sreynen) [reply docs/HOWTO--server-backend.md]: Just realized we can just use petitions.moveon.org as the example now that the API is live there.

File: docs/HOWTO--server-backend.md (hunk 1 above)
Comment (schuyler1d) [reply docs/HOWTO--server-backend.md]: is that actually necessary at all?  I think BASE_APP_PATH's default is your local env, which should be fine, right?

File: docs/HOWTO--server-backend.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# How to Develop with a Server Backend
+
+When running locally, you should set the following environment variables:
+
+```
+export API_URI="https://petitions.moveon.org"
+export BASE_APP_PATH="/path/to/your/local/mop-frontend/local/"
```
Comment (schuyler1d) [inline docs/HOWTO--server-backend.md]: Is BASE_APP_PATH necessary?  if folks are running with just `npm run dev` I think it should set it correctly by default.

**Pull request merged**